lib/gui/components/drive-selector/templates/drive-selector-modal.tpl.html

```diff
@@ -50,7 +50,17 @@ <h4 class="list-group-item-heading">{{ drive.description }} - {{ drive.size | gi
 </div>
 
 <div class="modal-footer">
-  <button class="button button-primary button-block"
+  <div class="modal-footer-warning"
```

Comment:
What about `modal-footer-label`? Its a warning because we think of it that way, but I imagine this could be used for something else in the future.

Changing it to a label also means that `div` doesn't make much sense anymore, although its not a big deal. If you're motivated to make such change, I guess you can make it something like a `<p>`.